### DISCUSSION TITLE
Cannot be accessed by other devices on the local network

### DISCUSSION THREAD

**Discussion by onlysunjun** [langgenius/dify#5633] [General]
Configured on a Mac using the docker-compose.yml file. After configuration, it can only be accessed from the local machine. Accessing from other devices on the local network results in an ERR_ADDRESS_UNREACHABLE error

**Comment by santime27** [comment]
What IP are you accessing with?. Cause you must utilize IP of the device where you deploy dify

**Reply by onlysunjun** [reply]
I found the reason. It might be because I used Surge on the machine and configured it to take over the gateway. After switching to another machine, this problem was resolved. Thx.
